tests/workspace/test_cli.py

```diff
@@ -144,3 +144,25 @@ def test_import_dir_helper(tmpdir):
             assert any([ca[0][2] == WorkspaceLanguage.SQL \
                     for ca in import_workspace.call_args_list])
 
+
+def test_import_dir_rstrip(tmpdir):
+    """
+    Copy from directory ``tmpdir`` with structure as follows
+    - a (directory)
+      - test-py.py (python)
+    """
+    os.makedirs(os.path.join(tmpdir.strpath, 'a'))
+    with open(os.path.join(tmpdir.strpath, 'a', 'test-py.py'), 'wb') as f:
+        f.write('print 1 + 1')
```

Comment:
where did you check the content of this file?

Reply:
We don't check the content of the file. The test just checks to see if the paths have their extensions stripped correctly.

Reply:
If that is the case, you don't need any content here. You could just put `pass` here.

Reply:
Ok let's do that.